### PR TITLE
Verbose installs

### PR DESCRIPTION
**Pull request by JackMc**
Addresses #10

### CODE REVIEW

File: datacats/cli/install.py [hunk 1]
```diff
@@ -71,9 +73,13 @@ def install_all(environment, clean):
 
     for s in ['ckan'] + sorted(srcdirs):
         write('Installing ' + s)
-        environment.install_package_develop(s)
+        c_id = environment.install_package_develop(s)
+        if verbose and c_id:
+            print '\n'.join(container_logs(c_id, "all", True, None))
```
Comment (wardi) [inline datacats/cli/install.py]: we should be using the generator to get the terminal output as it arrives, not all at once after completion. Some of these installs are really slow.

Comment (JackMc) [inline datacats/cli/install.py]: :+1:

**Comment by JackMc** [comment]
Ready for CR

**Comment by JackMc** [comment]
Doesn't do anything - duly noted!

**Pull request merged**